src/javascript/_common/base/livechat.js

```diff
@@ -1,28 +1,35 @@
-const { init }     = require('@livechat/customer-sdk');
-const BinarySocket = require('./socket_base');
-const ClientBase = require('./client_base');
+const BinarySocket  = require('./socket_base');
+const ClientBase    = require('./client_base');
+const TrafficSource = require('../../app/common/traffic_source');
+const licenseID     = require('../utility').lc_licenseID;
 
 const LiveChat = (() => {
-    const licenseID = 12049137;
-    const clientID = '66aa088aad5a414484c1fd1fa8a5ace7';
-    let session_variables = { loginid: '', landing_company_shortcode: '', currency: '', residence: '', email: '' };
+    const utm_data = TrafficSource.getData();
+    const utm_source = TrafficSource.getSource(utm_data) || '';
+    const utm_campaign = utm_data.utm_campaign || '';
+    const utm_medium = utm_data.utm_medium || '';
+    let session_variables = { is_logged_in: false, loginid: '', landing_company_shortcode: '', currency: '', residence: '', email: '', utm_source: utm_source || '', utm_medium: utm_medium || '', utm_campaign: utm_campaign || '' };
```

Comment:
utm source, campaign, and medium already fallback to '' in the declaration above. I think the repeated fallback '' for them is not needed here.